docs/developer_guide.md

```diff
@@ -296,6 +296,27 @@ make ovms_builder_image
 make run_unit_tests
 ```
 
+To run unit tests verifying integration with Intel GPUs, add `RUN_GPU_TESTS=ON` parameter:
+```
+make run_unit_tests RUN_GPU_TESTS=ON
```

Comment:
RUN_TESTS parameter takes values '1' or '0'. I think it would be more consistent to use the same approach here

Reply:
@atobiszei was there a reason you defined `ON`? if not, i agree, we can change to 1/0

Reply:
done